test/test.hs

```diff
@@ -70,6 +92,336 @@ main = defaultMain tests
 runSubSet :: String -> IO ()
 runSubSet p = defaultMain . applyPattern p $ tests
 
+data EVMToolTrace =
+  EVMToolTrace
+    { evmPc :: Int
+    , evmOp :: Int
+    , evmGas :: W256
+    , evmMemSize :: Integer
+    , evmDepth :: Int
+    , evmRefund :: Int
+    , evmOpName :: String
+    , evmStack :: [W256]
+    , evmError :: Maybe String
```

Comment:
I don't think we need these prefixes

Reply:
Todo `_error` then when new GHC comes out, we can fix.